packages/rs-drive-abci/src/execution/validation/state_transition/state_transitions/documents_batch/data_triggers/triggers/dpns/v0/mod.rs

```diff
@@ -88,13 +94,14 @@ pub fn create_domain_data_trigger_v0(
         .get_bool_at_path(ALLOW_SUBDOMAINS)
         .map_err(ProtocolError::ValueError)?;
 
-    let mut result = DataTriggerExecutionResult::default();
-    let full_domain_name = if normalized_parent_domain_name.is_empty() {
-        normalized_label.to_string()
+    let full_domain_name = if parent_domain_name.is_empty() {
+        label.to_string()
     } else {
-        format!("{normalized_label}.{normalized_parent_domain_name}")
+        format!("{normalized_label}.{parent_domain_name}")
```

Comment:
why no longer normalized?

Reply:
We want people to see the original domains that he registered in the error message.